addon/components/bread-crumbs.js

```diff
@@ -63,7 +64,15 @@ export default Component.extend({
     const routes = routeNames.slice(0, index + 1);
 
     if (routes.length === 1) {
-      return `${name}.index`;
+      let path = `${name}.index`;
+
+      if (this._lookupRoute(path)) {
+        return path;
+      } else {
+        warn(`[ember-crumbly] \`route:${path}\` was not found, falling back to \`route:${name}\``);
```

Comment:
I don't think this warning is necessary, might get annoying for apps that will only ever have flat routes